cmd/earthly/app/run.go

```diff
@@ -259,6 +244,49 @@ func (app *EarthlyApp) run(ctx context.Context, args []string) int {
 				"You can login using the command:\n"+
 				"  docker login%s", registryName, registryHost)
 			return 1
+		case grpcErrOK && grpcErr.Code() == codes.PermissionDenied && buildMinutesRegex.MatchString(grpcErr.Message()):
+			msg := grpcErr.Message()
+			matches, _ := stringutil.NamedGroupMatches(msg, buildMinutesRegex)
+			if len(matches["msg"]) > 0 {
+				msg = matches["msg"][0]
+			}
+			msg = fmt.Sprintf("%s (%s)", msg, stringutil.Title(billing.Plan().Tier))
+			app.BaseCLI.Console().VerboseWarnf(err.Error())
+			app.BaseCLI.Logbus().Run().SetGenericFatalError(time.Now(), logstream.FailureType_FAILURE_TYPE_OTHER, msg)
+			switch billing.Plan().Tier {
+			case billingpb.BillingPlan_TIER_LIMITED_FREE_TIER:
+				app.BaseCLI.Console().HelpPrintf("Visit your organization settings to verify your account\nand get 6000 free build minutes per month: %s\n", billing.GetBillingURL(app.BaseCLI.CIHost(), app.BaseCLI.Flags().OrgName))
+			case billingpb.BillingPlan_TIER_FREE_TIER:
+				app.BaseCLI.Console().HelpPrintf("Visit your organization settings to upgrade your account: %s\n", billing.GetUpgradeURL(app.BaseCLI.CIHost(), app.BaseCLI.Flags().OrgName))
+			}
+			return 1
+		case grpcErrOK && grpcErr.Code() == codes.PermissionDenied && maxSatellitesRegex.MatchString(grpcErr.Message()):
+			msg := grpcErr.Message()
+			matches, _ := stringutil.NamedGroupMatches(msg, maxSatellitesRegex)
+			if len(matches["msg"]) > 0 {
+				msg = matches["msg"][0]
+			}
+			msg = fmt.Sprintf("%s %s", stringutil.Title(billing.Plan().Tier), msg)
+			app.BaseCLI.Console().VerboseWarnf(err.Error())
+			app.BaseCLI.Logbus().Run().SetGenericFatalError(time.Now(), logstream.FailureType_FAILURE_TYPE_OTHER, msg)
+			switch billing.Plan().Tier {
+			case billingpb.BillingPlan_TIER_LIMITED_FREE_TIER:
+				app.BaseCLI.Console().HelpPrintf("Visit your organization settings to verify your account\nfor an option to launch more satellites: %s\nor consider removing one of your existing satellites (`earthly sat rm <satellite-name>`)", billing.GetBillingURL(app.BaseCLI.CIHost(), app.BaseCLI.Flags().OrgName))
```

Comment:
Add billing URL here too?

Reply:
Oh - we are... I thought you missed one message but I just didn't see it. Ignore me :-)